scenarios/13_Scouting.cfg

```diff
@@ -808,8 +836,10 @@
             type=Peasant
             x=18
             y=6
+            # It's always rubbed me the wrong way that this is "Matthews Eagle Eye" rather than "Matthew Eagle's-Eye" but it's probably too late to change it now:
```

Comment:
Idk it sounds right to me? As the original I mean.

Reply:
when "Matthews" has an "s" it's usually a last name (e.g. "Chris Matthews") but here it seems to be being used as a first name? idk...

Reply:
Oh I see. Well you can change the name if you like.

Reply:
maybe later; I'll just clarify the comment with [efba779](https://github.com/nemaara/A_New_Order/pull/43/commits/efba77972af8c9bcc1fc309bcc70c2a6748d2016) for now...